### PR TITLE
Open Source Marathon title Glitch Effect

### LINKED ISSUE #3: I'm interested in incorporating a glitch effect into our open-source page.
I'm pleased to present a demonstrative 🎥 video showcasing the changes: [Video Link](https://drive.google.com/file/d/1TEyghpYvn88yKCfF-d0TWX-Pya8cXX8D/view?usp=sharing)

I assure you that these modifications won't compromise the page's responsiveness or text formatting.

I appreciate your consideration in this regard.

### PR DESCRIPTION
**Pull request by Rajveer07**

### CODE REVIEW

File: events/open-source-marathon.html [hunk 1]
```diff
@@ -31,6 +31,183 @@
...
+}
+
+.glitch:before {
+   content: attr(data-text);
+   position: absolute;
+   top: 0;
+   left: -2px;
+   width: 100%;
+   color: 50px;
+   background-image: url("/assets/img/grain-dark.png");
```
Comment (PradyumnaKrishna) [inline events/open-source-marathon.html]: What about `color` having 50px property?
I think `background-image` is not necessary here, you can just remove it.

File: events/open-source-marathon.html [hunk 2]
```diff
@@ -31,6 +31,183 @@
...
+}
+
+.glitch:after {
+   content: attr(data-text);
+   position: absolute;
+   top: 0;
+   left: 2px;
+   width: 100%;
+   color: 50px;
+   background-image: url("/assets/img/grain-dark.png");
```
Comment (PradyumnaKrishna) [inline events/open-source-marathon.html]: Same applies here as well.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
Looks good, left some comments.

File: events/open-source-marathon.html (hunk 2 above)
Comment (Rajveer07) [reply events/open-source-marathon.html]: color should be color:#ffffff;   yes we actually don't need backgroud-Image.  in both cases glitch:before and glitch:after   Should I again make pull request or is there ianother method to correct ? I think I got it . Done 🟢

**Review by PradyumnaKrishna — APPROVED** [review]
Well done @Rajveer07! LGTM

**Pull request merged**